index.html

```diff
@@ -229,10 +229,10 @@ <h3 class="hero-text hero-text-white">
                 If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
             </h3>
 
-            <form action="" class="contact-form">
-                <input type="text" name="" id="" placeholder="Yeremias" >
-                <input type="email" name="" id="" placeholder="natajayanj14@gmail.com" >
-                <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here" ></textarea>
+            <form action="https://formspree.io/f/xjvdwrqb" method="POST" class="contact-form">
+                <input required type="text" name=""  id="" placeholder="Ingresa tu nombre" maxlength="30" >
+                <input required type="email" name="" id="" placeholder="Ingresa tu correo" >
+                <textarea name="coment" cols="30" rows="10" placeholder="Ingresa tus comentarios" maxlength="500"></textarea>
                 <input class="btn" type="submit" value="Get in touch">
```

Comment:
😀 From what I noticed here, it seems the submit button has a little `box-shadow` design **on hover** from the template, if this is true, kindly implement it as such 😉.

- Your design
<img width="123" alt="wrong" src="https://user-images.githubusercontent.com/41163571/220482204-967aef9d-91f9-4ba8-9924-e35e7dfb8211.png">


- Figma design
<img width="113" alt="right" src="https://user-images.githubusercontent.com/41163571/220482232-9d8eb763-9285-429e-b0e7-78c579472a2c.png">

Reply:
![image](https://user-images.githubusercontent.com/50673336/220485405-7d39b4e2-5aa6-4bec-8d08-9200fa6bf4c3.png)

the box shadow is already there